src/components/logic/calculator.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import ButtonBox from '../calculator-components/buttonbox';
+import Screen from '../calculator-components/screen';
+import { operate } from './operate';
+
+const Calculator = () => {
+  const [calc, setCalc] = useState({
+    operator: '',
+    operand1: 0,
+    operand2: 0,
+    result: 0,
+    screenValue: '',
+  });
+
+  return (
+    <div className="wrapper">
+      <Screen value={calc.screenValue} />
+      <ButtonBox calc={calc} setCalc={setCalc} operate={operate} />
+    </div>
+  );
+};
+export default Calculator;
```

Comment:
- [ ] [REQUIRED] To make work easy and concentrate on how to work with React events, you are advised to use the already developed math logic needed to make the Calculator component functional. Kindly ensure that you use this file as recommended [calculate.js](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js). Please copy the file into your logic directory then analyze the file and be sure you understand what they do.

- Kindly refer to the project requirement:
<img width="924" alt="Screenshot 2023-04-27 at 17 22 51" src="https://user-images.githubusercontent.com/58692590/234927595-43010649-6ac8-4daf-a927-e829a5eb8177.png">